fda-model/compute/ImpactAreaScenarioSimulation.cs

```diff
@@ -101,9 +101,9 @@ private void AddEADKeys(ConvergenceCriteria convergenceCriteria)
         {
             foreach (UncertainPairedData uncertainPairedData in _damage_category_stage_damage)
             {
-                _impactAreaScenarioResults.ConsequenceResults.AddConsequenceResultObject(uncertainPairedData.CurveMetaData.DamageCategory, uncertainPairedData.CurveMetaData.AssetCategory, convergenceCriteria, _impactAreaID);
+                _impactAreaScenarioResults.ConsequenceResults.AddNewConsequenceResultObject(uncertainPairedData.CurveMetaData.DamageCategory, uncertainPairedData.CurveMetaData.AssetCategory, convergenceCriteria, _impactAreaID);
```

Comment:
It is best to remove namespaces.

Reply:
I don't see any namespaces at this location in the file. did a quick search in this file and cleaned out a few.